tvaroglu/relational_rails
# I2 pt ar update

What I did:
- completed user stories 15-23 with exception of #21
- refactored tests to use `before each`
- added records to seed file

Roadblocks:
- user story #21

Tests:
- 6 pending tests
- 96.47% coverage
- tests written for user story 21; incomplete

None:
- Not stoked about how some click buttons are titled (really long when referencing park/trail names in button); refactor if time
- Need to complete user story 21
- review and refactor if time
- ensure sufficient test coverage
- add more links to pages for usability
